core/src/main/scala/kafka/server/BrokerToControllerChannelManager.scala

```diff
@@ -319,7 +345,7 @@ class BrokerToControllerRequestThread(
     val requestIter = requestQueue.iterator()
     while (requestIter.hasNext) {
       val request = requestIter.next
-      if (currentTimeMs - request.createdTimeMs >= retryTimeoutMs) {
+      if (currentTimeMs - request.createdTimeMs >= requestThreadRetryTimeoutMs) {
```

Comment:
This is the only place this particular retry timeout is used.  It is not clear to me how the semantics of this timeout and the network client timeout differ and why the values are used differently (specifically, AlterParticionManager currently sets Long.MAX_VALUE for this timeout but uses `controller.socket.timeout.ms` for the network client timeout).